src/nems_subscription_delete/app.py

```diff
@@ -1,45 +1,27 @@
-import json
-
-
-# import requests
-
-
-def lambda_handler(event, context):
-    """Sample pure Lambda function
-
-    Parameters
-    ----------
-    event: dict, required
-        API Gateway Lambda Proxy Input Format
-
-        Event doc: https://docs.aws.amazon.com/apigateway/latest/developerguide/set-up-lambda-proxy-integrations.html#api-gateway-simple-proxy-for-lambda-input-format
-
-    context: object, required
-        Lambda Context runtime methods and attributes
-
-        Context doc: https://docs.aws.amazon.com/lambda/latest/dg/python-context-object.html
-
-    Returns
-    ------
-    API Gateway Lambda Proxy Output Format: dict
-
-        Return doc: https://docs.aws.amazon.com/apigateway/latest/developerguide/set-up-lambda-proxy-integrations.html
-    """
-
-    # try:
-    #     ip = requests.get("http://checkip.amazonaws.com/")
-    # except requests.RequestException as e:
-    #     # Send some context about this error to Lambda Logs
-    #     print(e)
-
-    #     raise e
-
-    return {
-        "statusCode": 200,
-        "body": json.dumps(
-            {
-                "message": "hello world",
-                # "location": ip.text.replace("\n", "")
-            }
-        ),
-    }
+from uuid import UUID
+
+from aws_lambda_powertools import Logger
+from aws_lambda_powertools.utilities.typing import LambdaContext
+from fhir.resources.operationoutcome import OperationOutcome, OperationOutcomeIssue
+
+_LOGGER = Logger()
+
+
+@_LOGGER.inject_lambda_context(log_event=True)
+def lambda_handler(event: dict, context: LambdaContext):
+    try:
+        UUID(event["pathParameters"]["id"])
+    except ValueError:
```

Comment:
This is all good - but I don't spy any unit tests for this behaviour (it's a behaviour we won't want to loose in regression, hence it needs a unit test!)